src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/ha-cluster-support.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-05-14
+modDate: 2024-05-14
+title: HA Cluster Support
+description: How to install/update the agent when running Octopus in an HA Cluster
+navOrder: 60
+---
+
+## Octopus Deploy HA Cluster
+
+Similarly to Polling Tentacles, the Kubernetes Agent must have a URL for each individual node in the HA Cluster so that it receive commands from all clusters. These URLs must be provided when registering the Agent or some deployments may fail depending on which node the tasks are executing.
```

Comment:
```suggestion
Similarly to Polling Tentacles, the Kubernetes agent must have a URL for each individual node in the HA Cluster so that it receive commands from all clusters. These URLs must be provided when registering the agent or some deployments may fail depending on which node the tasks are executing.
```